### PR TITLE
Make refactoring and new features

### PR DESCRIPTION
**Pull request by AmsTaFFix**
- change add subscribers to add multiple subscribers always
- now library can handle all features of POST /Api/Subscribers resource
- changed documentation
- add new fields to Subscriber POST request
- refactored Response, add new method isEmpty
- change LICENSE, composer.jsom, README.md
- fix tests
- add isEmpty method to ResponseInterface
- change architecture, now we can handle 100% functionality of subscribers add/edit
- move classes into new namespaces

### CODE REVIEW

File: src/Model/SubscribersPostRequest/Identifier.php [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+     *
+     * @return string Custom subscriber ID
+     */
+    public function getCustomSubscriberId(): string
+    {
+        if (!$this->matchingMode->equals(MatchingMode::CUSTOMER_SUBSCRIBER_ID())) {
+            throw new ExpertSenderApiException(
+                sprintf(
+                    'Can\'t use getCustomSubscriberId method, because Matching mode is "%s", if you want use '
+                    .'custom subscriber id as identifier create object with static method createCustomSubscriberId',
```
Comment (liderman) [inline src/Model/SubscribersPostRequest/Identifier.php]: Violation of standard PSR

File: src/Model/SubscribersPostResponse/SubscriberData.php [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+/**
+ * Subscriber info after add/edit
+ *
+ * @author Nikita Sapogov <sapogov.n@citilink.ru>
+ */
+class SubscriberData
+{
+    /**
+     * @var string
```
Comment (liderman) [inline src/Model/SubscribersPostResponse/SubscriberData.php]: Please add a description

File: src/Model/SubscribersPostResponse/SubscriberData.php [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+     */
+    private $email;
+
+    /**
+     * @var int
```
Comment (liderman) [inline src/Model/SubscribersPostResponse/SubscriberData.php]: Please add a description

File: src/Model/SubscribersPostResponse/SubscriberData.php [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+     */
+    private $id;
+
+    /**
+     * @var bool
```
Comment (liderman) [inline src/Model/SubscribersPostResponse/SubscriberData.php]: Please add a description

File: src/Model/SubscribersPostResponse/SubscriberData.php [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+     */
+    private $wasAdded;
+
+    /**
+     * @var bool
```
Comment (liderman) [inline src/Model/SubscribersPostResponse/SubscriberData.php]: Please add a description

File: src/Model/SubscribersPostResponse/SubscriberData.php [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+     */
+    private $wasIgnored;
+
+    /**
+     * Конструктор.
+     *
+     * @param string $email
```
Comment (liderman) [inline src/Model/SubscribersPostResponse/SubscriberData.php]: Please add a description

File: src/Model/SubscribersPostResponse/SubscriberData.php [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+     * @param int $id
+     * @param bool $wasAdded
```
Comment (liderman) [inline src/Model/SubscribersPostResponse/SubscriberData.php]: `$wasAdded` - `was`? It seems to me better `is` or `has`

File: src/Response.php [hunk 8]
```diff
@@ -106,34 +87,91 @@ public function getHttpStatusCode(): int
...
+     */
+    public function getSimpleXml(): \SimpleXMLElement
+    {
+        if ($this->simpleXml === null) {
+            if ($this->isEmpty()) {
+                throw new ParseResponseException(
+                    'Response is empty, it\'s impossible to create simpleXML element. Maybe it is error, or response '
+                    . 'does not have content. You can use method isEmpty before trying to get SimpleXML'
+                );
+            }
```
Comment (liderman) [inline src/Response.php]: Add an empty line after `}`

Comment (AmsTaFFix) [inline src/Response.php]: fixes

**Pull request merged**